modules/470-chrony/template_tests/module_test.go

```diff
@@ -72,15 +72,19 @@ var _ = Describe("Module :: chrony :: helm template ::", func() {
 			registrySecret := f.KubernetesResource("Secret", "d8-chrony", "deckhouse-registry")
 
 			chronyDaemonSetTest := f.KubernetesResource("DaemonSet", "d8-chrony", "chrony")
-			chronyMasterDeploymentTest := f.KubernetesResource("Deployment", "d8-chrony", "chrony-master")
+			chronyMasterDaemonsetTest := f.KubernetesResource("DaemonSet", "d8-chrony", "chrony-master")
 
 			Expect(namespace.Exists()).To(BeTrue())
 			Expect(registrySecret.Exists()).To(BeTrue())
 
 			Expect(chronyDaemonSetTest.Exists()).To(BeTrue())
-			Expect(chronyMasterDeploymentTest.Exists()).To(BeTrue())
+			Expect(chronyMasterDaemonsetTest.Exists()).To(BeTrue())
 			Expect(chronyDaemonSetTest.Field("spec.template.spec.containers.0.env").String()).To(MatchJSON(`
         [
+		  {
+            "name": "PATH",
+            "value": "/opt/chrony-static/bin"
```

Comment:
Why not in /opt/deckhouse/bin?

Reply:
it's inside container, /opt/deckhouse/bin is path on the node